cypress/e2e/3-Register-page/1-filling.cy.js

```diff
@@ -1,23 +1,22 @@
 describe('Registration Form Test 1', () => {
-    it('should fill out the registration form and check the checkbox', () => {
- 
+  it('should fill out the registration form and check the checkbox', () => {
       cy.visit('/auth/register');
 
-      cy.get('[name="fullName"]').type('Maryna Vasylyk');
-      cy.get('[name="email"]').type('marynav@example.com');
-      cy.get('[name="password"]').type('password123');
-      cy.get('[name="rePass"]').type('password123');
+      cy.get('body').within(() => {
```

Comment:
It is bad practice, better to rich to container in some form. f.e you can use "nb-register"